src/feedback-rating/types.ts

```diff
@@ -0,0 +1,13 @@
+export interface FeedbackRatingProp {
+    imgSrc?: string;
+    description?: string;
+    buttonLabel?: string;
+    ratings: number;
+    onRatingsChange: (value: number) => void;
```

Comment:
for consistency, could we stick to singular `rating` and `onRatingChange` here?

Reply:
updated (removed 's')